hibernate-ogm/playsport-demo/src/main/java/org/hibernate/ogm/demo/playsport/rest/exception/ExceptionDto.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2017 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+/**
+ * Created by fabio on 18/08/2017.
+ */
+package org.hibernate.ogm.demo.playsport.rest.exception;
+
+public class ExceptionDto {
```

Comment:
Could you add some more javadoc to such helper classes? I'm not using REST very often so I'm not sure what these are for. I guess people studying the demo will want to understand this as well.